frontend/public/components/pod-logs.jsx

```diff
@@ -5,59 +5,62 @@ import { Dropdown, LoadingInline, ResourceLog, ResourceName } from './utils';
 
 // Component to container dropdown or conatiner name if only one container in pod.
 const ContainerDropdown = ({currentContainer, containers, kind, onChange}) => {
-  const resourceName = (container) => <ResourceName name={container.name || <LoadingInline />} kind={kind} />;
+  const resourceName = (container) => {
+    const name = container.name || <LoadingInline />;
+    return <ResourceName name={name} kind={kind} />;
+  };
   const dropdownItems = _.mapValues(containers, resourceName);
-  return <Dropdown className="btn-group" items={dropdownItems} title={resourceName(currentContainer)} onChange={onChange} />;
+  return <Dropdown
+    className="btn-group"
+    items={dropdownItems}
+    title={resourceName(currentContainer)}
+    onChange={onChange} />;
 };
 
 export class PodLogs extends React.Component {
   constructor(props) {
     super(props);
-    this._selectContainer = this._selectContainer.bind(this);
-
     this.state = {
       containers: [],
-      currentContainer: null,
+      index: 0 // Index for the currently selected container
     };
   }
 
+  _selectContainer(i) {
```

Comment:
If you call the variable `index`, you can write this as:

`this.setState({index});`